components/tyk-gateway/templates/deployment-gw-repset.yaml

```diff
@@ -96,6 +96,26 @@ spec:
             value: "{{ .Values.gateway.control.containerPort }}"
           {{- end }}
 
+          - name: TYK_GW_STORAGE_SSLINSECURESKIPVERIFY
+            value: "{{ .Values.global.redis.sslInsecureSkipVerify }}"
+          
+          # Redis TLS configurations
+          {{- if .Values.global.redis.useSSL }}
+          - name: TYK_GW_STORAGE_SSLMAXVERSION
+            value: "{{ .Values.global.redis.sslMaxVersion }}"
+          - name: TYK_GW_STORAGE_SSLMINVERSION
+            value: "{{ .Values.global.redis.sslMinVersion }}"
+          {{- end }}
+
+          {{- if .Values.global.redis.sslCAFile }}
+          - name: TYK_GW_STORAGE_SSLCAFILE
+            value: "{{ .Values.global.redis.sslCAFile }}"
+          - name: TYK_GW_STORAGE_SSLCERTFILE
+            value: "{{ .Values.global.redis.sslCertFile }}"
+          - name: TYK_GW_STORAGE_SSLKEYFILE
+            value: "{{ .Values.global.redis.sslKeyFile }}"
+          {{- end }}          
```

Comment:
users can provide a Cert File and a Key file without providing a CA file, so we should modify this `if statement`.